plugins/flytekit-modin/flytekitplugins/modin/schema.py

```diff
@@ -0,0 +1,152 @@
+import os
+import typing
+from typing import Type
+
+from flytekit import FlyteContext
+from flytekit.configuration import sdk
+from flytekit.extend import T, TypeEngine, TypeTransformer
+from flytekit.models.literals import Literal, Scalar, Schema
+from flytekit.models.types import LiteralType, SchemaType
+from flytekit.types.schema import LocalIOSchemaReader, LocalIOSchemaWriter, SchemaEngine, SchemaHandler
+from flytekit.types.schema import SchemaFormat
+from flytekit.types.schema.types import FlyteSchemaTransformer
+
+from modin import pandas
+
+
+class ParquetIO(object):

```

Comment:
i do not think you need to reimplement all of this.

I think `modin.pandas` can be similar to `pyspark.sql.DataFrame`. As modin.pandas.DataFrame is actually a disparate object?

If so, we can simply implement the [ModinDataFrameSchemaReader like](https://github.com/flyteorg/flytekit/blob/28153cc787bdd529f9f6050c393687715476153d/plugins/flytekit-spark/flytekitplugins/spark/schema.py#L13)
and  [ModinDataFrameSchemaWriter like](https://github.com/flyteorg/flytekit/blob/28153cc787bdd529f9f6050c393687715476153d/plugins/flytekit-spark/flytekitplugins/spark/schema.py#L31) and finally a Transformer like
[this](https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-spark/flytekitplugins/spark/schema.py#L50)
https://github.com/flyteorg/flytekit/blob/master/plugins/flytekit-spark/flytekitplugins/spark/schema.py